.rhiza/tests/integration/test_book_targets.py

```
@@ -0,0 +1,146 @@
+    lines = content.splitlines()
+    new_lines = []
+    for line in lines:
+        if line.strip().startswith("LOGO_FILE"):
+            # Comment out the line
+            new_lines.append(f"# {line}")
+        else:
+            new_lines.append(line)
+    makefile.write_text("\n".join(new_lines))
+
+    # Dry-run the book target - it should still be valid
+    result = subprocess.run([MAKE, "-n", "book"], cwd=git_repo, capture_output=True, text=True)  # nosec
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)